### PR TITLE
Tests for increasing codecov

### PR DESCRIPTION
**Pull request by eleallegue**
Still woring on issue #33 , few tests remaining

### CODE REVIEW

**Comment by eleallegue** [comment]
Still working ont the InsertDataTest class for testint the insertion of data.

File: src/test/java/parser/ExcelParserTest.java [hunk 1]
```diff
@@ -97,9 +98,10 @@ public void testEmptyRow() {
 
 	@Test
 	public void testCorrect() {
-		new ExcelParser("src/test/resources/test.xlsx");
+		ExcelParser ex = new ExcelParser("src/test/resources/test.xlsx");
 		assertThat(logger.getLoggingEvents(), is(asList(info("Starting parsing..."), info("Agent added"),
 				info("Agent added"), info("Agent added"), info("Finish parsing..."))));
+		assertTrue(ex.getFileFullPath().equals("src/test/resources/test.xlsx"));
```
Comment (thewilly) [inline src/test/java/parser/ExcelParserTest.java]: O would not only check that the logger says it’s been created but also that the content of the excelparser contains all the agents in the excel.

Comment (eleallegue) [reply src/test/java/parser/ExcelParserTest.java]: Yes sure! I will implement that test in the InsertDataTest class which I will implement right away!

Comment (eleallegue) [reply src/test/java/parser/ExcelParserTest.java]: Excuse me, yes, in this test file is where I will implement that, I got confused with the list of agentsToInsert.

Comment (thewilly) [reply src/test/java/parser/ExcelParserTest.java]: hahah I already thought you would figure it out 😸 .

**Pull request merged**